src/interfaces/message.ts

```diff
@@ -0,0 +1,5 @@
+export interface MessageModel {
+  role: string
```

Comment:
Could we align this with the other places where roles are defined? In fact there's an almost identical class in the project.